src/site/resources/styleguides/google-java-style-20180523/javaguide.html

```diff
@@ -249,7 +251,7 @@ <h2 id="s4-formatting">4 Formatting</h2>
 <a name="braces"></a>
 <h3 id="s4.1-braces">4.1 Braces</h3>
 
-<h4 id="s4.1.1-braces-always-used">4.1.1 Braces are used where optional</h4>
+<h4 id="s4.1.1-braces-always-used">4.1.1 Use of optional braces</h4>
 
```

Comment:
will be handled by: https://github.com/checkstyle/checkstyle/issues/14908

Reply:
Closed/merged.